### ISSUE TITLE
A search bar without enabling search results in a blank page.

### ISSUE THREAD

**Issue by damithc** [MarkBind/markbind#1571]
If a search bar is used in a top navigation bar but built-in search is not enabled in the site.json, the whole page shows up blank. While this is caused by a user error, perhaps the behavior can be more user-friendly?

This screenshot shows how the page looks when the search is enabled (as expected):
![image](https://user-images.githubusercontent.com/1673303/120878546-cea97600-c5ef-11eb-83d5-9edd9f6091a5.png)

When search is disabled in site.json, the whole page is blank.

**Comment by tlylt** [comment]
Hi prof @damithc ,

I have done some preliminary testing and could not reproduce the error. Not sure if the issue has already been resolved.

---
Steps that I have taken:
- `markbind init testsite`
- `cd testsite`
- `markbind serve`
=> Works fine as `enableSearch : true` by default
- add `enableSearch : true` in `site.json`
=> Works fine
- change to `enableSearch : false` in `site.json`
=> search box does not respond to typing, console is logging error about `searchCallback` undefined (happens only sometimes, perhaps due to reloading), but page loads fine and stopping and re issuing `markbind serve` still works.

I think if what I described are the intended steps and corresponding response, it's quite fault-tolerant.
